pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[project]
+name = "aws-sdk-python"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+    "mkdocs-awesome-pages-plugin>=2.10.1",
+]
```

Comment:
Flagging this so we don't forget about it. If we end up needing this, it should be removed from both `dependencies` and the `dev` dependency group. It should just get added with the others in `docs`